src/rules/first-name-rules.ts

```diff
@@ -1,5 +1,4 @@
 export const FirstNameRules: Array<Function> = [
-  v => !!v || 'First name is required',
-  v => /^(?=.*\d)|(?=.*[a-z])|(?=.*[A-Z])/.test(v) || 'First name is required', // Contains at least a single character
+  v => !!v.trim() || 'First name is required',
```

Comment:
You need to verify that v is not null before you call trim on its instance.

Reply:
Idea:
```
v => (!v || v.trim().length < 1) || 'First name is required`
```

Reply:
Optional chaining seems to work

Reply:
I think your new rules works, too.

Reply:
Sorry, i had two quick commits. I put in the optional chaining immediately after :)